ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -204,6 +247,56 @@ private void terminateCluster(final Command command) {
     log.info("The KSQL server was terminated.");
   }
 
+  public double checkCommandRunnerStatus() {
+    if (currentCommand == null) {
+      return 1;
+    }
+
+    return Duration.between(currentCommand.right, Instant.now()).toMillis()
+        < COMMAND_RUNNER_HEALTH_TIMEOUT.toMillis() ? 1 : 0;
+  }
+
+  private void configureCommandRunnerStatusMetric() {
```

Comment:
I'd consider pulling the metric out into another class like `CommandRunnerStatusMetric` with all the boilerplate for defining a metric:

```
class CommandRunnerStatusMetric {
    final Metrics metrics;
    final MetricName name;

    public CommandRunnerStatus(
            final Metrics metrics,
            final Supplier<Status> statusSupplier  // computes the status based on the current command/time
    ) {
        this.metrics = metrics;
        setup(metrics, statusSupplier);
    }

    void close() {
        metrics.removeMetric(metricName);
    }
}
```

Reply:
Moved the metric code out to separate file.